docs/api/README.md

```diff
@@ -176,13 +176,17 @@ const store = new Vuex.Store({ ...options })
 
 -  `subscribe(handler: Function, options?: Object): Function`
 
-  Subscribe to store mutations. The `handler` is called after every mutation and receives the mutation descriptor and post-mutation state as arguments:
+  Subscribe to store mutations. The `handler` is called after every mutation and receives the mutation descriptor and post-mutation state as arguments.
+  The `subscribe` method will return an `unsubscribe` function, which should be called when the subscription is no longer needed. For example, when unregistering a Vuex module or before destroying a Vue component.
```

Comment:
I think we should be a little more explicit about the use cases here. Because people might get confused about Vuex module, or Vue Component mentioned here since there is no example nor description provided formerly about such use cases. So how about we make it like this.

First of all, let's move `unsubscribe` feature description at the last of the section (replace "To stop subscribing, call the returned unsubscribe function." text). Then add:

---

The `subscribe` method will return an `unsubscribe` function, which should be called when the subscription is no longer needed. For example, you might subscribe to a Vuex Module, and then later, you want to unregister the module, or you might call `subscribe` method inside Vue Component and then destroy the component later. In these cases, you should remember to unsubscribe the subscription manually.